conans/client/graph/graph_builder.py

```diff
@@ -56,14 +56,14 @@ def _resolve_deps(self, node, aliased, update, remote_name):
             if alias:
                 req.conan_reference = alias
 
-        if not hasattr(conanfile, "_evaluated_requires"):
-            conanfile._evaluated_requires = conanfile.requires.copy()
-        elif conanfile.requires != conanfile._evaluated_requires:
+        if not hasattr(conanfile, "_conan_eval_requires"):
```

Comment:
I'll keep the past perfect tense, for me `eval` has a meaning and `evaluated` a different one.